docs/user/reporting/reporting-troubleshooting.asciidoc

```diff
@@ -92,6 +92,31 @@ the first time Kibana starts when verbose logging is enabled.
 Whenever possible, a Reporting error message tries to be as self-explanatory as possible. Here are some error messages you might encounter,
 along with the solution.
 
+[float]
+==== `StatusCodeError: [version_conflict_engine_exception]`
+If you are running multiple instances of Kibana in a cluster, they share the work of executing report jobs to evenly distribute
+the work load. Each instance searches the reporting index for "pending" jobs that the user has requested. It is possible for
+multiple instances to find the same job in these searches. Only the instance that successfully updated the job status to
+"processing" will actually execute the report job. The other instances that unsuccessfully tried to make the same update will log
+something similar to this:
+
+```
```

Comment:
Use formatting here so that code is in a black box.  See the section of this page called "Verbose logs" for an example.